### ISSUE TITLE
[Bug] autotvm_matmul  Check failed:(code == RPCCode::kReturn)is false: code =kShutdown

### ISSUE THREAD

**Issue by MalaJeans** [apache/tvm#15151]
When using the official tutorial: autotvm_matmul_x86.py, the tuning trials is set to 10,
 and the matrix size is set to N, L, M = 2043*2043*2043. An error occurred during the execution as follows:
 Check failed:(code == RPCCode::kReturn)is false: code =1
 Check failed:(code == RPCCode::kReturn)is false: code =kShutdown
 How can I solve this problem?

 My environment is:
 Ubuntu 20.04.6

Thanks!

**Comment by Johnson9009** [comment]
It seems the RPC server timeout when executing, so try to increase the "session_timeout".

**Comment by MalaJeans** [comment]
> 
Change the session_timeout of rpc.connect(), right？
In which file rpc.connect() needs to be modified?
Thanks!

**Comment by MalaJeans** [comment]
> It seems the RPC server timeout when executing, so try to increase the "session_timeout".
Change the session_timeout of rpc.connect(), right？
In which file rpc.connect() needs to be modified?
Thanks!

**Comment by MalaJeans** [comment]
@Johnson9009 
Change the session_timeout of rpc.connect(), right？
In which file rpc.connect() needs to be modified?
Thanks!

**Comment by Johnson9009** [comment]
> @Johnson9009 Change the session_timeout of rpc.connect(), right？ In which file rpc.connect() needs to be modified? Thanks!

Yes, just try increase the session_timeout of rpc.connect, if you use RPC tracker, it is the session_timeout of "request".

```python3
    tracker = rpc.connect_tracker(tracker_host, tracker_port)
    remote = tracker.request(key, priority=0, session_timeout=60)
```

**Comment by Johnson9009** [comment]
Please change the tile of your issue, so others can know what the problem about and help you.

**Comment by MalaJeans** [comment]
OK,  thanks！I will have a try!

**Comment by MrGouYou** [comment]
> > @Johnson9009 Change the session_timeout of rpc.connect(), right？ In which file rpc.connect() needs to be modified? Thanks!
> 
> Yes, just try increase the session_timeout of rpc.connect, if you use RPC tracker, it is the session_timeout of "request".
> 
> ```python
>     tracker = rpc.connect_tracker(tracker_host, tracker_port)
>     remote = tracker.request(key, priority=0, session_timeout=60)
> ```

Actually, i am running with 'LocalRunner', and  also get similar problem. 
![image](https://github.com/apache/tvm/assets/12627651/68d0c280-8ac0-4f80-98f3-b4a14a4ae3bb)

![image](https://github.com/apache/tvm/assets/12627651/635b1f8e-fa61-4961-a525-cde7f72f39c6)
